Content.Server/Cargo/Systems/PricingSystem.cs

```diff
@@ -108,6 +112,25 @@ private void CalculateStackPrice(EntityUid uid, StackPriceComponent component, r
         args.Price += stack.Count * component.Price;
     }
 
+    private void CalculateSolutionPrice(EntityUid uid, SolutionContainerManagerComponent component, ref PriceCalculationEvent args)
+    {
+        var price = 0f;
+
+        foreach( ( _ ,var solution) in component.Solutions)
+        {
+            if (solution.Contents.Count() == 0)
+                continue;
+
+            foreach(var reagent in solution.Contents)
```

Comment:
```suggestion
            foreach (var reagent in solution.Contents)
```

Unnecessary when you're just iterating over it.